dspace-api/src/main/java/org/dspace/statistics/Dataset.java

```diff
@@ -54,8 +54,8 @@ public Dataset(int rows, int cols) {
         initRowLabels(rows);
     }
 
-    public Dataset(float[][] matrix) {
-        this.matrix = (String[][]) ArrayUtils.clone(matrix);
+    public Dataset(String[][] matrix) {
```

Comment:
Why has this type been modified from float to String?

Reply:
@benbosman : because of the line immediately following that one... notice how the previous code takes in a `float` double array, clones it and directly casts it into a `String` double array.  

My compiler complained this cast was not allowed in Commons Lang v3.  I initially worked to find a replacement, but couldn't find an easy solution. After realizing that this `Dataset` constructor is unused, I decided to change the type instead.

If we ever find a real need for a `float` based constructor, we can revisit this. But currently we don't use or need it.